_posts/2020/2021-01-07-how-to-animate-a-svg-with-css.md

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
Could you please change the PR title to a more meaningful title? Otherwise it's hard to organize the different PRs and know what to expect in which PR :)